netci.groovy

```diff
@@ -349,4 +350,47 @@ def osShortName = ['Windows 10': 'win10',
     }
 }
 
+// **************************
+// Define Linux ARM Emulator testing. This creates a per PR job which
+// cross builds native binaries for the Emulator rootfs.
+// NOTE: To add Ubuntu-ARM cross build jobs to this code, add the Ubuntu OS to the
+// OS array, branch the steps to be performed by Ubuntu and the Linux ARM emulator
+// based on the OS being handled, and handle the triggers accordingly
+// (the machine affinity of the new job remains the same)
+// **************************
+[true].each { isPR ->
+    ['Debug', 'Release'].each { configurationGroup ->
+        ['LinuxARMEmulator'].each { os ->
+            def osGroup = osGroupMap[os]
+            def newJobName = "${os.toLowerCase()}_${configurationGroup.toLowerCase()}"
+            def arch = "arm-softfp"
+
+	    // Setup variables to hold emulator folder path and the rootfs mount path
+	    def armemul_path = '/opt/linux-arm-emulator'
+	    def armrootfs_mountpath = '/opt/linux-arm-emulator-root'
+
+            def newJob = job(Utilities.getFullJobName(project, newJobName, isPR)) {
+                steps {
+                    // Call the arm32_ci_script.sh script to perform the cross build of native corefx
+                    shell("./scripts/arm32_ci_script.sh --emulatorPath=${armemul_path} --mountPath=${armrootfs_mountpath} --buildConfig=${configurationGroup.toLowerCase()}")
+                }
+            }
+
+            // The cross build jobs run on Ubuntu. The arm-cross-latest version
+            // contains the packages needed for cross building corefx
+            Utilities.setMachineAffinity(newJob, 'Ubuntu', 'arm-cross-latest')
+
```

Comment:
Ubuntu14.04.. The Ubuntu in Utilities.groovy exists to just handle the case when we were only using 14.04 and had used osname as Ubuntu.. Going forward we will keep it specific as 14.04 or 16.04..